resources/views/product/partials/gallery/thumbnails.blade.php

```diff
@@ -0,0 +1,39 @@
+<div v-if="images.length > 1" class="flex mt-3 gap-2">
+    <button
+        v-for="(image, imageId) in images.slice(0, {{ max(config('rapidez.frontend.product_gallery_thumbnails.xs'), config('rapidez.frontend.product_gallery_thumbnails.sm'), config('rapidez.frontend.product_gallery_thumbnails.md'), config('rapidez.frontend.product_gallery_thumbnails.lg'), config('rapidez.frontend.product_gallery_thumbnails.xl')) + 1 }})"
```

Comment:
`max(config('rapidez.frontend.product_gallery_thumbnails'))` should be enough, but... I don't like the config for such a small part separated from the template. Maybe just define them on top of here?
```blade
@php($breakpoints = ['xl' => 7, 'lg' => 5, 'md' => 4, 'sm' => 3, 'xs' => 4])
```
And use that here; `max($breakpoints)` and through the template

Reply:
But if we _only_ want to change the breakpoints and everything else stays the same, then do we have to override this file in our project just to change breakpoints instead of changing it in the config file?